validator/client/runner.go

```diff
@@ -92,11 +93,14 @@ func run(ctx context.Context, v Validator) {
 						"pubKey": fmt.Sprintf("%#x", bytesutil.Trunc(id[:])),
 						"role":   role,
 					})
+
 					switch role {
 					case pb.ValidatorRole_BOTH:
 						go v.SubmitAttestation(slotCtx, slot, id)
+						v.SubmitAggregateAndProof(ctx, slot, id)
```

Comment:
Can this be in a go routine so it doesn't block proposing a block?

Reply:
Also, wouldn't this be part of SubmitAttestation?

Reply:
Ha. Yeah. Was going to add it next PR when we implement the logic to `Wait till 3/4 of the slot and submit aggregated attestation and proof` I'll add it now :)